src/editpage/components/assistance/SettingsModal.vue

```diff
@@ -174,6 +191,17 @@ export default {
       this.updateConfigurationString();
     },
 
+    exportLogs: function() {
+      const {dialog} = require('electron').remote;
+      const options = {defaultPath: 'logs.zip'};
+      const targetDir = dialog.showSaveDialogSync(options);
+      const electron = require('electron');
+      const sourceDir = (electron.app || electron.remote.app).getPath(
+          'userData') + '\\activity-logs';
```

Comment:
I think we already store/use this path somewhere, maybe we can make this a common function?

Reply:
Or at least make some spot where we define `activity-logs` to ensure we can easily change the name.

Reply:
I also found the path in /src/background.js. So I should make a common function to return the path when called?

Reply:
Yes, just to make sure the paths don't drift away from each other.